external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,560 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
+<!-- TODO: uri_override not in schema -->
+<!-- | `uri_override`      | Override the uri created automatically for the column (Optional) (Advanced)                                                                                                                                          | `tidy_data.csv#uri_safe_column_header_from_csv`                                  | -->
 
-| **field name**   | **description**                                                                                                           | **default value**                                                                |
-|------------------|---------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|
-| `type`           | The type of the column (Required)                                                                                         | *dimension*                                                                      |
-| `label`          | The title of the column (Optional)                                                                                        | The capital case of the header in the csv file with spaces replacing underscores |
-| `description`    | A description of the contents of the column (Optional)                                                                    | *none*                                                                           |
-| `from_existing`  | The uri of the resource for reuse/extension (Optional)                                                                    | *none*                                                                           |
-| `definition_uri` | A uri of a resource to show how the column is created/managed (i.e. a uri of a PDF explaining a list of units) (Optional) | *none*                                                                           |
+### Examples of dimension configuration
 
-The `from_existing` value when set provides the basis of linked data; it allows csvcubed to generate additional RDF-hints to allow users to discover how the `tidy_data.csv` links to other data semantically.
+The [Sweden at Eurovision data set](../../quick-start/designing-csv.md#adding-your-data) consists of four dimensions - `Year`, `Entrant`, `Song` and `Language`. Examples of how these dimensions could be configured are as follows.
 
-## Dimension Configuration
+```json
+{ ...
+   "columns": {
+      "Year": {
+         "from_template": "year",
+         "label": "Competition year",
+         "code_list": "true"
```

Comment:
`"code_list": true` is the default for years already. Also, booleans shouldn't be in quotation marks in JSON.